### DISCUSSION TITLE
No hair?

### DISCUSSION THREAD

**Discussion by Derphead** [ErinaSugino/Starbound-SBR-Mod#109] [Trouble Shooting] [answered]
Is my hairstyle supposed to dissapear when having sex? I become a baldy :( Tried with both human and Neki character.

**Comment by ErinaSugino** [comment]
Again, log file.

**Comment by Derphead** [comment]
How do I do that? :D

**Reply by ErinaSugino** [reply]
`/storage/starbound.log`

**Reply by Derphead** [reply]
[starbound.log](https://github.com/ErinaSugino/Starbound-SBR-Mod/files/15444374/starbound.log)

**Comment by ErinaSugino** [comment, marked as answer]
Why your Neki turn bald human is answered quite easily - you don't have the Neki SxB compat mod.
Why your humans turn bald is because... The game says your hair doesn't exist?
You use modded hair, so maybe they do some funky shenanigans that I don't know of, but it's not where Sexbound expects it to be.

**Reply by Derphead** [reply]
You're right, didn't know about the neki SxB mod, thanks ;)
